sdks/python/apache_beam/runners/portability/sdk_container_builder.py

```diff
@@ -149,7 +149,6 @@ def invoke_docker_build_and_push(self, container_image_name):
       _LOGGER.info("Building sdk container, this may take a few minutes...")
       now = time.time()
       subprocess.run(['docker', 'build', '.', '-t', container_image_name],
-                     capture_output=True,
```

Comment:
How about 
`subprocess.run([ ], stdout=subprocess.PIPE)`
Woud this capture stdout and print STDERR?

Reply:
If there is an build error, how will users see it?

Reply:
Not capturing any output should help. But may be noisy. We can consider to capture the output but print it only if something goes wrong. 
Leaving the change as is SGMT for now.